src/odo.ts

```diff
@@ -752,11 +765,41 @@ export class OdoImpl implements Odo {
     }
 
     public async executeInTerminal(command: string, cwd: string = process.cwd(), name = 'OpenShift'): Promise<void> {
-        const cmd = command.split(' ')[0];
-        const toolLocation = await ToolsConfig.detect(cmd);
-        const terminal: Terminal = WindowUtil.createTerminal(name, cwd);
-        terminal.sendText(toolLocation === cmd ? command : command.replace(cmd, `"${toolLocation}"`).replace(new RegExp(`&& ${cmd}`, 'g'), `&& "${toolLocation}"`), true);
+        const process = await this.spawn(command, cwd);
+        const terminal: Terminal = WindowUtil.createPseudoTerminal(name);
+        terminal.sendText('Pushing component to the cluster');
```

Comment:
is this function exclusively used for pushing things to the cluster?

Reply:
@dgolovin If we run any other command like "List catalog components" or "List catalog services", it sends the same text to the terminal.

Reply:
@fbricon yes, in this POC only. It just a test how to use pseudo terminal and show output in terminal view. It has pros and cons. With pseudo terminal we can now detect start and finish of executed process, but it comes at price of running process with spawn and basically using terminal as a view to show output. There is no way to type and execute commands in it.